capstone/src/main/java/com/google/univiz/CollegeData.java

```diff
@@ -0,0 +1,76 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+/** CollegeData is a class representing a single university/college/institution. */
+@AutoValue
+abstract class CollegeData {
+
+  abstract int id();
```

Comment:
I suspect you'll want this to be `CollegeId`, which is introduced [here](https://github.com/googleinterns/univiz/blob/1d1061804b2bc9003da2aae7e9b850c6fe3c3823/capstone/src/main/java/com/google/univiz/api/CollegeId.java).
```suggestion
  abstract CollegeId id();
```